libs/common/src/types/mongo-db-models/facility.ts

```diff
@@ -0,0 +1,44 @@
+import { ObjectId } from 'mongodb';
+import { UnixTimestamp } from '../date';
+import { Currency } from '../currency';
+
+/**
+ * Type of the mongo db "facilities" collection
+ *
+ * This type is likely incomplete and should be added
+ * to as and when new properties are discovered
+ */
+export type Facility = {
+  _id: ObjectId;
+  dealId: ObjectId;
+  type: string;
+  hasBeenIssued: boolean;
+  name: string;
+  shouldCoverStartOnSubmission: boolean;
+  coverStartDate: UnixTimestamp | null;
+  coverEndDate: UnixTimestamp | null;
```

Comment:
just a reminder as talked about on slack, this is stored as a date, and I reckon it's mongodb storing it as a string that secretly is a date (`$date` field), and it may or may not be parsable as a `number` straight off the bat

Reply:
Turns out it is either a `UnixTimestampString` or a `Date` based on the staging database. Fortunately, any operations which use this date (multiplying it by 1000 usually) work whether the value is a string or a date in this case